packages/process/src/node/process-backend-contribution.ts

```diff
@@ -16,12 +16,12 @@
 
 import { injectable, inject, named } from 'inversify';
 import { ILogger } from '@theia/core/lib/common';
-import { TerminalProcess, ProcessManager } from '@theia/process/lib/node';
-import { terminalsPath } from '../common/terminal-protocol';
+import { ProcessManager } from '.';
+import { processesPath } from '../common/process-protocol';
 import { MessagingService } from '@theia/core/lib/node/messaging/messaging-service';
 
 @injectable()
-export class TerminalBackendContribution implements MessagingService.Contribution {
+export class ProcessBackendContribution implements MessagingService.Contribution {
```

Comment:
For my own understanding, why was the rename necessary?
Note, breaking changes should be added to the changelog.

Reply:
I find it hard to know the balance between a) avoiding breaking changes, and b) keeping the code clean.  In this particular case , as I understand it, 'terminal' processes are a subset of all processes.  It would be confusing if it were called 'Terminal' when in fact it now applies not just to terminal processes but to all processes.

Reply:
I understand what you mean, I guess a breaking change note would be enough then :)